includes/dbFacile.php

```diff
@@ -114,9 +118,13 @@ function dbUpdate($data, $table, $where = null, $parameters = array()) {
 
 	// need field name and placeholder value
 	// but how merge these field placeholders with actual $parameters array for the WHERE clause
-	$sql = 'UPDATE `' . $table . '` set ';
+	$sql = 'UPDATE `' . $table . '` SET ';
 	foreach($data as $key => $value) {
-                $sql .= "`".$key."` ". '=:' . $key . ',';
```

Comment:
Yeah those NOW() calls work ok and dbFacile is a standard library from elsewhere so it's easier if we can just update that with little issue.